src/Core/IEPlugin/IEPluginInit.cc

```diff
@@ -32,10 +32,12 @@
 #include <Core/IEPlugin/SimpleTextFileToMatrix_Plugin.h>
 #include <Core/IEPlugin/EcgsimFileToMatrix_Plugin.h>
 #include <Core/IEPlugin/PointCloudField_Plugin.h>
-#include <Core/IEPlugin/IgbFileToMatrix_Plugin.h>
+#include <Core/IEPlugin/IGBFileToMatrix_Plugin.h>
```

Comment:
@kgillette Can you make this change? Otherwise I have to make a copy of your branch, close this PR, and push it myself.